Watchman.Discord.UnitTests/AntiSpam/SpamDetectorsStrategiesTests.cs

```diff
@@ -65,5 +69,66 @@ public void DuplicatesDetector_ShouldNotDetectSpam(string messageContent1, strin
             // Assert
             Assert.That(spamProbability, Is.EqualTo(SpamProbability.None));
         }
+
+        [Test]
+        [TestCase(SpamProbability.Low, "not capslock", "not capslock", "not capslock", "not capslock", "not capslock", "CAPSLOCK")]
+        [TestCase(SpamProbability.Medium, "ALSO CAPSLOCK abc", "not capslock", "not capslock", "not capslock", "not capslock", "CAPSLOCK")]
+        [TestCase(SpamProbability.Sure, "ALSO CAPSLOCK abc", "ANOTHER CAPSLOCK", "NOT capslock", "not capslock XDD", "not capslock", "CAPSLOCK")]
+        public void CapslockDetector_ShouldDetectSpam(SpamProbability exceptedSpamProbability, string m1, string m2, string m3, string m4, string m5, string m6) // not using params to force 6 messages
+        {
+            var spamDetectorsTestsService = new SpamDetectorsTestsService<CapslockDetectorStrategy>();
+            var spamProbability = spamDetectorsTestsService.GetSpamProbability(isUserSafe: false, m1, m2, m3, m4, m5, m6);
+
+            // Assert
+            Assert.That(spamProbability, Is.EqualTo(exceptedSpamProbability));
+        }
+
+        [Test]
+        [TestCase("not capslock", "not capslock", "not capslock", "not capslock", "not capslock", "not capslock")]
+        [TestCase("REST API", "not capslock", "not capslock", "not capslock", "NOT CApslock", "XDDD")]
+        [TestCase("not capslock", "not capslock", "not capslock", "not capslock", "REST API", "XDDD")]
+        [TestCase("not capslock", "not capslock", "not capslock", "not capslock", "REST API", "NOT CApslock")]
+        public void CapslockDetector_ShouldNotDetectSpam(string m1, string m2, string m3, string m4, string m5, string m6) // not using params to force 6 messages
```

Comment:
nie rozumiem o co chodzi w tym komentarzu.

Reply:
poza tym zamiast pisać m1, m2 itd. to nie lepiej wysłać tablicę messages? Wtedy będzie tylko jeden parametr.

Reply:
chodzi o to, że chcę wymusić żeby zawsze było 6 wiadomości
jeśli byłoby to jako params to przypadkiem można by wpisać za mało albo za dużo wiadomości w testCase

Reply:
coś myślę, że tego nie trzeba pisać w komentarzu, bo to wynika z kodu.

Reply:
właśnie lepiej zostawić, bo w niektórych miejscach jest params więc tutaj będzie uzasadnienie